tests/manual/performance/init.md

```diff
@@ -0,0 +1,17 @@
+# Performance: Editor init time
+
+**! Test with console off, in incognito mode !**
+
+This performance test measures how long editor initializes for various kinds of documents. Each data set presents a "heavy" document case, which takes substantial amount of time to load.
+
+The test starts automatically after you open this page.
+
+By default, the test goes through all data sets and performs 10 tries for each. Default run will take around 10 minutes. You can narrow it down using the test config in test source.
+
+The status message will inform you about current progress.
+
+After each single editor initialization, the initialization time is stored and the page is refreshed, until the whole suite is done.
+
+After all tests are done, the results will be inserted into the clipboard, in a format that can be directly pasted to a spreadsheet. Allow the browser to use the clipboard when prompted. Results are also displayed in the console.
```

Comment:
It didn't prompt me. Then I realized the clipboard access is disabled for the page 
<img width="635" alt="image" src="https://github.com/user-attachments/assets/07ab9e27-25ac-460b-8fbb-05f8ff7dad58">
in the settings.

I realized this is because I use a fake `fake.ckeditor.com` domain for development and the browser treats it as unsafe (no https). It's worth mentioning that you have to run this test on `localhost`.